airbyte-integrations/connectors/destination-timeplus/destination_timeplus/spec.json

```diff
@@ -0,0 +1,34 @@
+{
+  "documentationUrl": "https://docs.timeplus.com",
+  "supported_destination_sync_modes": ["overwrite", "append"],
+  "supportsIncremental": true,
+  "supportsDBT": false,
+  "supportsNormalization": false,
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Destination Timeplus",
+    "type": "object",
+    "required": [
+      "endpoint",
+      "apikey"
+    ],
+    "additionalProperties": false,
+    "properties": {
+      "endpoint": {
+        "title": "Endpoint",
```

Comment:
Timeplus provides fully-managed cloud as well as on-prem deployment. For public cloud, it can be https://us.timeplus.cloud and https://cloud.timeplus.com.cn. For on-prem deployments, the endpoint can be any hosted domain.